### PR TITLE
UAT: Deployment template

### PR DESCRIPTION
**Pull request by AbhishekA1509**
This PR includes:
- SelectPicker small variant
- fixing gaps in App Metrics
- add variant in popup menu type for AppConfig

### CODE REVIEW

File: src/Shared/Components/SelectPicker/utils.ts [hunk 1]
```diff
@@ -165,6 +185,10 @@ export const getCommonSelectStyle = <OptionValue, IsMulti extends boolean>({
     }),
     dropdownIndicator: (base, state) => ({
         ...base,
+        ...getIconSize(size),
+        display: 'flex',
+        alignItems: 'center',
+        flexShrink: '0',
```
Comment (eshankvaish) [inline src/Shared/Components/SelectPicker/utils.ts:191]: let's use Map instead for size/dimensions like we did in Button

File: src/Shared/Helpers.tsx [hunk 2]
```diff
@@ -821,3 +821,20 @@ export const getDefaultValueFromType = (value: unknown) => {
...
+        case ComponentSizeType.medium:
+            return 3
+        case ComponentSizeType.large:
+            return 4
+        case ComponentSizeType.xl:
+            return 5
+        default:
+            return 0
+    }
+}
```
Comment (eshankvaish) [inline src/Shared/Helpers.tsx:840]: this won't be required

**Review by eshankvaish — CHANGES_REQUESTED** [review]
Reviewed only the SelectPicker part

**Review by eshankvaish — APPROVED** [review]

File: src/Shared/Components/SelectPicker/constants.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    [ComponentSizeType.large]: '13px',
+}
+
+export const SELECT_PICKER_ICON_SIZE_MAP: Record<SelectPickerProps['size'], { width: string; height: string }> = {
+    [ComponentSizeType.small]: { width: '12px', height: '12px' },
+    [ComponentSizeType.medium]: { width: '16px', height: '16px' },
+    [ComponentSizeType.large]: { width: '16px', height: '16px' },
+}
+
+export const SELECT_PICKER_CONTROL_SIZE_MAP: Record<SelectPickerProps['size'], string> = {
```
Comment (RohitRaj011) [inline src/Shared/Components/SelectPicker/constants.ts:16]: can make this more type safe instead of string use CSSProperties

**Review by RohitRaj011 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**